docs/docs-ref-autogen/excel-async/overview.md

```diff
@@ -1,28 +1,32 @@
 ---
 title: Office Scripts API reference 
-description: 'An overview of the Office Scripts JavaScript APIs'
-ms.date: 06/16/2020
+description: 'An overview of the Office Scripts Async JavaScript APIs'
+ms.date: 06/17/2020
 ---
 
-# Office Scripts API reference
+# Office Scripts Async API reference
 
-The Office Scripts API lets you automate common tasks in Excel on the web. Use this reference documentation to learn more about the classes, methods, and other types available for your scripts. All the objects accessible through Office Scripts can be found in the table of contents on the left of the page.
+The Office Scripts Async API supports older scripts made during the Office Scripts preview phase. Use this reference documentation to learn more about the classes, methods, and other types used by these older scripts. All the objects accessible through Office Scripts can be found in the table of contents on the left of the page.
+
+> [!IMPORTANT]
+> We strongly recommend creating new scripts with the standard Office Scripts APIs. If you're making or editing a new script, please switch to the [non-async version](?view=office-scripts-async) of the APIs.
```

Comment:
What page are you linking to? Also, is the view correct?

Reply:
Yeah, wrong view... This is linking to the current page, but with a different moniker.